apps/system/js/utility_tray.js

```diff
@@ -614,10 +701,33 @@ window.UtilityTray = {
     }
   },
 
-  _shouldPrevent: function ut_shouldPrevent(currentTarget) {
-    var targeted = [this.grippy, this.statusbarIcons, this.topPanel];
-    return targeted.some(function(target) {
-      return target === currentTarget;
-    });
+  _shouldPrevent: function ut_shouldPrevent(target) {
+    return [
+      this.statusbarIcons,
```

Comment:
Do we need to prevent in the statusbarIcons if we don't listen on events anymore? (https://github.com/mozilla-b2g/gaia/pull/30525/files#diff-690ac4f812d69b66c808d3e430042345L59)